source/renderer/app/stores/CurrencyStore.js

```diff
@@ -0,0 +1,131 @@
+// @flow
+import { action, observable, computed, runInAction } from 'mobx';
+import Store from './lib/Store';
+import {
+  CURRENCY_REQUEST_RATE_INTERVAL,
+  getLocalizedCurrency,
+  getLocalizedCurrenciesList,
+  getCurrencyFromCode,
+} from '../config/currencyConfig';
+import type { Currency, LocalizedCurrency } from '../types/currencyTypes.js';
+
+export default class CurrencyStore extends Store {
+  @observable currencyIsFetchingList: boolean = false;
+  @observable currencyIsFetchingRate: boolean = false;
+  @observable currencyIsAvailable: boolean = false;
+  @observable currencyIsActive: boolean = false;
+  @observable currencyList: Array<Currency> = [];
+  @observable currencySelected: ?Currency = null;
+  @observable currencyRate: ?number = null;
+  @observable currencyLastFetched: ?Date = null;
```

Comment:
I think we could remove `currency…` at the beginning of these store properties. And reference it directly from the store like this: `currency.rate` in the components

Reply:
Nice catch! Done ✅